daemon/start.go

```diff
@@ -176,19 +176,35 @@ func (daemon *Daemon) containerStart(container *container.Container, checkpoint
 		return err
 	}
 
-	err = daemon.containerd.Create(context.Background(), container.ID, spec, createOptions)
+	ctx := context.TODO()
+	for i := 0; i < 2; i++ {
+		err = daemon.containerd.Create(context.Background(), container.ID, spec, createOptions)
```

Comment:
sleep a tiny bit before retry?

Reply:
We don't need to since the cleanup is happening synchronously, except I suppose as a small backoff. But I don't think it's necessary.

Reply:
Actually, this is the wrong place. This should be handled on restore.... will post a fix.